Defra.Cdp.Backend.Api/Endpoints/TestSuiteEndpoint.cs

```diff
@@ -48,9 +48,11 @@ static async Task<IResult> FindTestSuites(string name, IDeployableArtifactsServi
     }
 
     private static async Task<IResult> FindAllTestSuites(IDeployableArtifactsService deployableArtifactsService,
-    [FromServices] ITestRunService testRunService, CancellationToken cancellationToken)
+        HttpContext httpContext, [FromServices] ITestRunService testRunService, CancellationToken cancellationToken)
```

Comment:
iirc query params can just go in as nullable params and the controller will sort them out (there's annotations if you want the name to be different from the variable).

``` FindAllTestSuites(
       IDeployableArtifactsService deployableArtifactsService,
       string? teamId,
        [FromServices] ITestRunService testRunService, 
        CancellationToken cancellationToken
   )```